### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Antoine Leresche & Robin Forestier

### DISCUSSION THREAD

**Discussion by Forestierr** [heig-vd-dai-course/heig-vd-dai-course#484] [Show and tell]
Flappy bird game implemented via a client-server app.

The main idea is to create a simple game dynamic with just 1 player. 
Then, if time allows, we'd like to add a two-player mode where the second player places the pipes in real time.

Practical work repository is available at : https://github.com/Forestierr/dai-flappy-bird
Considered commit for evaluation:
https://github.com/Forestierr/dai-flappy-bird/commit/33463a3212f9870252a3e29e4e818bbe45deec81

**Comment by ludelafo** [comment]
OK for us, but your application should be able to manage multiple games.

Please be wary of the time, it can become hard quite easily.
